server/src/services/initialization/indexWorkspaceFolders.ts

```diff
@@ -83,7 +83,7 @@ export async function indexWorkspaceFolders(
       try {
         await foundProject.initialize();
       } catch (error) {
-        logger.error(`Error initializing ${foundProject.basePath}: ${error}`);
```

Comment:
Lets log the error in its entirety. So `logger.error(error)`, if it starts clogging up sentry I will suppress it with a later patch.